### PR TITLE
Issue #15449: partially excluded rule42 from google-java-format.sh

### PR DESCRIPTION
**Pull request by Zopsss**
#15449 

https://checkstyle.org/google_style.html#a4.2

### CODE REVIEW

File: .ci/google-java-format.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.j
     | grep -v "rule53camelcase" \
     | grep -v "rule522classnames" \
     | grep -v "rule41" \
-    | grep -v "rule42" \
+    | grep -v "rule42/ClassWithChainedMethods.java" \
```
Comment (Zopsss) [inline .ci/google-java-format.sh:18]: diff for ClassWithChainedMethods.java:

```diff
C:\OpenSource\checkstyle git:[15449-rule42]
git diff src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
diff --git a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
index 67fbdfbe8..202b2a53c 100644
--- a/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
+++ b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/ClassWithChainedMethods.java
@@ -9,15 +9,15 @@ public class ClassWithChainedMethods {
   public static void main(String[] args) {
     new ClassWithChainedMethods()
         .getInstance("string_one")
-    .doNothing("string_one".trim(), "string_two");
+        .doNothing("string_one".trim(), "string_two");:wa
     // violation above ''method call' child has incorrect indentation level 4, expected .* 8.'
 
     int length =
-    new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
+        new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
     // violation above ''new' has incorrect indentation level 4, expected .* 8.'
 
     int length2 =
-    new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
+        new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
     // violation above ''new' has incorrect indentation level 4, expected .* 8.'
   }

```

after formatted code test started failing so I had to grep it

Comment (romani) [reply .ci/google-java-format.sh:18]: Thanks  a lot. Good decision.

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**